pytorch_cnn_trainer/engine.py

```diff
@@ -358,6 +392,10 @@ def fit(
 
         print("Done Training, Model Saved to Disk")
 
+    history = {
+        "train": {"top1_acc": top1_acc, "top5_acc": top5_acc, "loss": loss},
+        "val": {"top1_acc": loss_v, "top5_acc": top5_acc_v, "loss": top1_acc_v},
```

Comment:
Here I have shuffled the value  for top1_acc_v and loss_v would commit this change in next pr

Reply:
Yeah that's a bad miss from our side, we should fix this up 😄